src/main/java/com/faforever/client/preferences/PreferencesService.java

```diff
@@ -181,11 +182,16 @@ public static void configureLogging() {
 
   /**
    * It may happen that the file is empty when the process is forcibly killed, so remove the file if that happened.
+   *
+   * @return true if the file was deleted
    */
-  private void deleteFileIfEmpty() throws IOException {
+  private boolean deleteFileIfEmpty() throws IOException {
     if (Files.size(preferencesFilePath) == 0) {
       Files.delete(preferencesFilePath);
+      preferences = new Preferences();
+      return true;
     }
+    return false;
```

Comment:
`else {
    return false;
}`

Reply:
no why???

Reply:
cause it's ten times more readable. No need to use the "abort" case here.

Reply:
we do this code style all over the application